mgmtd/mgmt_txn.c

```diff
@@ -2385,6 +2386,55 @@ int mgmt_txn_send_get_tree_oper(uint64_t txn_id, uint64_t req_id,
 	get_tree->simple_xpath = simple_xpath;
 	get_tree->xpath = XSTRDUP(MTYPE_MGMTD_XPATH, xpath);
 
+	if (content != GET_CONTENT_STATE) {
```

Comment:
So with flags this becomes `CHECK_FLAG(content, GET_DATA_FLAG_CONFIG)` which is much clearer IMO.

Reply:
Done.